README.md

```diff
@@ -4,19 +4,19 @@
 
 I2Cwrapper is a **generic modular framework for Arduino I2C target devices**. It supports AVR-based Arduinos (Uno, Nano etc.), ESP8266, and ESP32 platforms. 
 
-I2Cwrapper consists of an easily extensible firmware framework and a controller library. Together, they **take care of the overhead** necessary for implementing an I2C target device and let you concentrate on implementing the actual target functionality. Ready to use modules exist for **stepper motor** control, **servo motor** control, **digital/analog pin** control, and **ESP32 touch button** control (see below).
+I2Cwrapper consists of an easily extensible firmware framework and a controller library. Together, they **take care of the overhead** necessary for implementing an I2C target device and let you concentrate on implementing the actual target functionality. <replace ith Available Modules section?> Ready to use modules exist for **[stepper motor](#accelstepperi2c)** control via `AccelStepper`, **[servo motor](#servoi2c)** control via [`Servo` library](https://www.arduino.cc/reference/en/libraries/servo), **[digital/analog pin](#pini2c)** control, and **[ESP32 touch button](#esp32sensorsi2c)** control (see below).
```

Comment:
We could replace last sentence with the section below?

Reply:
Ok.